draft-ietf-quic-ack-frequency.md

```diff
@@ -68,7 +68,7 @@ normative:
 informative:
 
   Cus22:
-    title: "Reducing the acknowledgement frequency in IETF QUIC"
+    title: "Reducing the acknowledgment frequency in IETF QUIC"
```

Comment:
```suggestion
    title: "Reducing the acknowledegment frequency in IETF QUIC"
```

Not this one, it's somebody else's!

Reply:
Actually a quick Internet search tell me that they do use the "e" in the title. So this was correct.

Reply:
Lucas suggestion is correct, as you said it does have an 'e' in the title, and I just committed it to main.

Reply:
Ah right... got confused if we remove or at it... thanks!